jsx/StaticDataTable.js

```diff
@@ -49,6 +49,31 @@ StaticDataTable = React.createClass({
            'PageNumber' : 1
        });
     },
+    downloadCSV: function() {
+        var headers = this.props.Fields,
+            csvworker = new Worker('js/workers/savecsv.js');
```

Comment:
this should include loris.BaseURL. Otherwise it won't work from within a module..